### PR TITLE
Topic/analyze phenotypes page

### PR DESCRIPTION
**Pull request by lukasmueller**
Adds a tool to compare trials at /tools/trial/comparison/list (with list input) and /tools/trial/comparison/params (where trial names have to be specified by trial_name parameters)
Test is at t/selenium2/tools/trial_compare.t

### CODE REVIEW

File: lib/SGN/Controller/Analyze/Phenotypes.pm [hunk 1]
```diff
@@ -0,0 +1,26 @@
+
```
Comment (lukasmueller) [inline lib/SGN/Controller/Analyze/Phenotypes.pm]: Good catch. I think they are redundant.

File: mason/analyze/phenotypes.mas [hunk 2]
```diff
@@ -0,0 +1,179 @@
+<%args>
```
Comment (lukasmueller) [inline mason/analyze/phenotypes.mas]: fixed in cf34b12b6106615f377247d86210534b8d078a8a (removed the file). Saved a copy of the branch in topic/analyze_phenotypes_page_bryan.

File: mason/analyze/phenotypes.mas [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+    </td>
+  </tr>
+  </tbody>
+  </table>
+</form>
+</div>
+
+<&| /page/info_section.mas, title=>"Results" &>
+
+  <& /analyze/phenotype_results.mas &>
```
Comment (lukasmueller) [inline mason/analyze/phenotypes.mas]: The parent mason component was removed.

File: lib/SGN/Controller/Analyze/Phenotypes.pm (hunk 1 above)
Comment (lukasmueller) [inline lib/SGN/Controller/Analyze/Phenotypes.pm]: fixed in cf34b12b6106615f377247d86210534b8d078a8a (removed the file). Saved a copy of the branch in topic/analyze_phenotypes_page_bryan.

File: lib/SGN/Controller/AJAX/TrialComparison.pm [hunk 4]
```diff
@@ -0,0 +1,298 @@
...
+    print STDERR Dumper($data);
+    my ($fh, $tempfile) = $c->tempfile(TEMPLATE=>"compare_trials/trial_phenotypes_download_XXXXX");
+    foreach my $line (@$data) { 
+	my @columns = split "\t", $line;
+	my @quoted_columns = map { "\"$_\"" }  @columns;
+	my $csv_line = join ",", @quoted_columns;
+	print $fh $csv_line."\n";
+    }
+    my $temppath = $c->config->{basepath}."/".$tempfile;
+
```
Comment (lukasmueller) [inline lib/SGN/Controller/AJAX/TrialComparison.pm]: Fixed in fab7562270e9b78fcb39d66bfb1b39c42f7df961.

**Comment by isaak** [comment]
install required R packages: tidyr, ggplot2

**Comment by isaak** [comment]
installed R dependencies for all production and test sites.

**Pull request merged**